src/scripts/btma/annotator.py

```diff
@@ -30,12 +30,14 @@
 script_path = path.Path(__file__).parent.parent.absolute()
 
 omn_path = src_path / 'ontology' / 'edits'              
-csv_path = src_path / 'ontology' / 'exports'          
+csv_path = script_path / 'btma' / 'data'          
+db_path = script_path / 'btma' / 'database'
+exp_path = script_path / 'btma' / 'exports'
 owl_path = src_path / 'ontology' / 'imports'
 robot_path = script_path / 'btma' / 'robot.jar'
 
-def extract_IDs(module : list) -> list:
-    file = open(csv_path / (module + '.csv'))
+def extract_IDs(path) -> list:
```

Comment:
Note that when the prefix is specified to extract, as suggested below, the IDs will automatically be extracted in the form OEO:xxxxxxx so you can remove the split here, and just use ```if row[0].startswith('OEO'):
   ids.append(row[0])```

Reply:
Done